ember-liveagent/package.json

```
@@ -0,0 +1,132 @@
+{
+  "name": "ember-liveagent",
+  "version": "0.2.0",
+  "description": "Salesforce's live agent chat integration for Ember.js applications.",
+  "keywords": [
+    "ember-addon",
+    "salesforce",
+    "liveagent",
+    "chat"
+  ],
+  "homepage": "https://github.com/zonkyio/ember-liveagent#readme",
+  "repository": "https://github.com/zonkyio/ember-liveagent.git",
```

```suggestion
  "repository": {
    "type": "git",
    "url": "https://github.com/zonkyio/ember-liveagent.git",
    "directory": "ember-liveagent"
  },
```

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository